src/reanimated2/animation/timing.ts

```diff
@@ -65,6 +65,16 @@ export const withTiming = function (
 ): Animation<TimingAnimation> {
   'worklet';
 
+  if (userConfig?.easing) {
+    const { easing } = userConfig;
+    if (!('__workletHash' in easing)) {
```

Comment:
It seems that the solution doesn't work specifically for that one bezier object in 😅
https://github.com/software-mansion/react-native-reanimated/blob/main/src/reanimated2/Easing.ts#L226

We can evaluate the performance and determine whether this optimization is still necessary. If it doesn't significantly affect the performance, we can consider removing it.